projects/buttons/fastener/scripts/start-button.sh

```diff
@@ -136,6 +143,9 @@ tar xfz $DISTRO
 # check for existence (and executable-ness) of ./buttons-start.sh
 export FUSION_API_BASE; export FUSION_API_CREDENTIALS; export ADMIN_PASSWORD; export IP; ./buttons-start.sh
 
+# Add security.json file to script as environment variable to check existence.
+export SECURITY_JSON; ./buttons-start.sh
```

Comment:
Calling buttons-start.sh again?!   No, don't do that.    If you need to add it, add it to the line above with the other exports and call to buttons-start.